aws-ssm-maintenancewindowtask/src/test/java/software/amazon/ssm/maintenancewindowtask/CreateHandlerTest.java

```diff
@@ -1,54 +1,209 @@
 package software.amazon.ssm.maintenancewindowtask;
 
+import org.junit.jupiter.api.Assertions;
+import org.mockito.ArgumentMatchers;
+import software.amazon.awssdk.services.ssm.model.*;
+import software.amazon.cloudformation.exceptions.*;
 import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
 import software.amazon.cloudformation.proxy.Logger;
-import software.amazon.cloudformation.proxy.OperationStatus;
 import software.amazon.cloudformation.proxy.ProgressEvent;
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+
+import static junit.framework.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.eq;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import software.amazon.ssm.maintenancewindowtask.translator.ExceptionTranslator;
+import software.amazon.ssm.maintenancewindowtask.translator.request.RegisterTaskWithMaintenanceWindowTranslator;
+
+import java.util.function.Function;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.*;
+import static software.amazon.ssm.maintenancewindowtask.TestConstants.*;
```

Comment:
Wildcard (*) imports should be expanded to be only the imports that are needed. Here is how to turn off auto-correcting imports to wildcard imports: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

Reply:
Updated. Thank you!